lib/components/user/places/place-location-field.js

```diff
@@ -45,6 +47,17 @@ const StyledLocationField = styled(LocationField)`
       color: #999;
     }
   }
+  ${InputGroup} {
+    border: none;
+    width: 100%;
+  }
+  ${InputGroupAddon} {
+    display: none;
+  }
+  ${MenuItemList} {
+    margin-left: ${props => props.static ? '-50px' : '0px'};
+    position: absolute;
+  }
```

Comment:
Not sure where this is happening, but I noticed that the dropdown menu items are showing text underlined on hover here, whereas in the main location field they are not. Any idea why? Ideally we would have little difference visually.

Reply:
Not quite sure how Bootstrap applied extra styles... Fixed in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/436b93da0ef77b9c75628f92d29cf8313085d163 and https://github.com/opentripplanner/otp-react-redux/pull/280/commits/65c087f18c0cb4ef1cbdd84e8c4835a6825c1bcc.